spec/animatable_region_spec.coffee

```diff
@@ -1,3 +1,4 @@
+require './spec_helper'
```

Comment:
Do we need to require `spec_helper` manually? As far as I know it's automatically required by Karma.